test-resources/migrations/v0_14_2/tasks/pending/1700751070000-sweep_login_cache

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Why don't these files have `.json` extensions?

Reply:
no strong reason, except that we already had a database (and memory) store in mind, so the extension which seems more appropriate to filesystems was dropped along the way.

Addding it now means that we should either wipe the task queue on upgrade startup, or map the existing keys. So, I'd like to avoid it for now.